Add scroll buttons to tabs when they go beyond the available space (#4538)
marmelab/react-admin: docs/Show.md

```
{% endraw %}

> **Tip**: By default, `<TabbedShowLayout>` use MaterialUI scrollable buttons when there is too much Tabs to display. If you preferer the screen to expands with the tabs, use `scrollable={false}`.

Reviewer: ```suggestion
**Tip**: By default, `<TabbedShowLayout>` use MaterialUI scrollable buttons when there are too many Tabs to display. If you prefer the screen to expands with the tabs, use `scrollable={false}`.
```
